### DISCUSSION TITLE
I want to help

### DISCUSSION THREAD

**Discussion by MarcelRobitaille** [chipsenkbeil/distant#237] [General]
I want to help this project. I really believe it could make my workflow much better. Can I sponsor development? I only have a modest budget, but I would like to contribute. Is there something I could help work on? I have some experience with rust and with neovim plugin development in lua.

**Comment by chipsenkbeil** [comment]
Appreciate the kind offer :) I'd recommend holding onto your money for better needs.

In terms of the Rust code, I still need to complete the refactoring to be more pluggable and switching over to Russh. Once that's done, it'll be easier to contribute such as writing the docker integration, refining the CLI, etc.

You're more than welcome to contribute to either project. There are a variety of rough edges remaining, and I haven't even sat down to release a new video demonstrating searching, terminal usage, or any of the other features that have been developed for distant.nvim.

My recommendation would be to find a feature that doesn't work as you'd expect or is missing, and see if you can fix or introduce it into the plugin or Rust code.
